nservicebus/azure-storage-persistence/configuration_sections_azure_[,6.2].partial.md

```diff
@@ -0,0 +1,5 @@
+### Configuration with Configuration Section
```

Comment:
`configuration_sections_` are the same partial for different nuget aliases and versions. I've split them as ASP has no config section support. That's why it's partial is empty.

It should be rendered in a following way: depending on the selected version of the nuget alias for this component, only one partial should be selected.

If I use a component name in all partials this could be ambiguous when two nuget aliases shared a range of versions (I know it's not a case). Frankly speaking I wasn't aware of this possibility and thought that only nuget aliases can be used. Should I change something?